src/transform.js

```diff
@@ -8,37 +8,47 @@ import * as parents from "./parents.js";
  * This function will not modify the root node of the input AST.
  *
  * @param {object | object[]} node AST node or array of nodes
- * @param {Object.<string, function> | function(object, string, object?)} transformation A map of node types to callbacks, or a single callback that will be called for all node types
+ * @param {Object.<string, function> | function(object, string, object?) | (Object.<string, function> | function(object, string, object?))[]} transformations A map of node types to callbacks, or a single callback that will be called for all node types, or a list of either, which will be applied in order
  * @param {object} [o]
  * @param {string | string[] | function} [o.only] Only walk nodes of this type
  * @param {string | string[] | function} [o.except] Ignore walking nodes of these types
  * @returns {object | object[]} The transformation's return value on the root node(s) of the input AST, or the root node(s) if the transformation did not return a value
  */
-export default function transform (node, transformation, o) {
-	return _transform(node, transformation, o);
+export default function transform (node, transformations, o) {
+	if (!Array.isArray(transformations)) {
```

Comment:
What's the rationale for having this logic here and not in `_transform()`?

Reply:
Just put this here so that it doesn't have to make this check every recursive call inside `_transform`

Reply:
Fair 👍🏼